atom/browser/api/atom_api_web_contents.cc

```diff
@@ -640,6 +643,76 @@ bool WebContents::IsDevToolsOpened() {
   return managed_web_contents()->IsDevToolsViewShowing();
 }
 
+void WebContents::EnableDeviceEmulation(const base::DictionaryValue& dict) {
+  if (type_ == REMOTE)
+    return;
+
+  blink::WebDeviceEmulationParams params;
+
+  if (dict.HasKey("screenPosition")) {
+    std::string screenPosition;
```

Comment:
In C++ we use `screen_position` instead `screenPosition` for variable names, we follows Google C++ style guide in Electron and Chromium, the code in `blink` follows a different style guide though.